.github/workflows/triage_add_to_project.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           echo 'IS_COLLABORATOR='$(jq -r '.data.repository.collaborators.totalCount' collaborators.json) >> $GITHUB_ENV
       - uses: actions/add-to-project@v0.4.1
         # only add issues/prs from outside contributors to the project
-        if: ${{ env.IS_COLLABORATOR == 0 }}
+        if: ${{ env.IS_COLLABORATOR == 0 || github.event.repository.name == "cypress-support-internal" || github.event.pull_request.user.login == "github-actions[bot]" || github.event.issue.user.login == "github-actions[bot]" }}
```

Comment:
It feels weird to be exposing internal/private repos in this way. Is this workflow better suited to live else-where at this point in time?

Reply:
What way would you suggest?  What is your concern exactly?

Reply:
discussed in post scrum. change look fine 👍🏻